vm/procedures/src/requestProcedure.js

```diff
@@ -43,6 +49,13 @@ export const requestProcedure = (vm) => {
 				return /**@type any*/ (scriptProcedure)(vm.evm)(request)
 			case 'eth_blockNumber':
 				return /** @type any */ (blockNumberProcedure(vm.blockchain)(request))
+			case 'tevm_dump_state':
+				return /** @type any */ (dumpStateProcedure)(vm.stateManager)(request)
+			case 'tevm_load_state': {
+				// @ts-ignore
+				const stateManager = vm.stateManager
```

Comment:
I am not too happy about this mismatch in state manager types. What does @roninjin10 and coderabbit think? 

I would like to import TevmEvm from vm/vm/tevm but that would create a circular dependency
vm -> procedures -> vm

Reply:
This is why state is a seperate package from Tevm. Just pass in an import('@tevm/state').TevmStateManager explicitly as a param to the requestProcedure function.

To stay consistent you would make it take an object:

```typescript
 * @param {object} options
 * @param {import('@tevm/state').TevmStateManager  stateManager
 * @param {import('@ethereumjs/vm').VM vm
```

Reply:
I'm gonna update all the ones that only use statemanager to only rquire state manager in a future pr